README.md

```diff
@@ -4,16 +4,48 @@ Apache Pekko
 [![Nightly Builds](https://github.com/apache/incubator-pekko/actions/workflows/nightly-builds.yml/badge.svg)](https://github.com/apache/incubator-pekko/actions/workflows/nightly-builds.yml)
 [![Nightly Aeron Tests](https://github.com/apache/incubator-pekko/actions/workflows/nightly-builds-aeron.yml/badge.svg?branch=main)](https://github.com/apache/incubator-pekko/actions/workflows/nightly-builds-aeron.yml)
 
-Apache Pekko is an open-source framework for building applications that are concurrent, distributed, resilient and elastic.
-Pekko uses the Actor Model to provide more intuitive high-level abstractions for concurrency.
+> Current status: We are currently working on the 1.0.0 milestone. we will be initially releasing incubator-pekko [1.0.0 Milestone](https://github.com/apache/incubator-pekko/milestone/1) and [incubator-pekko-sbt-paradox](https://github.com/apache/incubator-pekko-sbt-paradox/).
+
+Apache Pekko is an open-source framework for building applications that are concurrent, distributed, resilient and elastic. It uses the Actor Model to provide more intuitive high-level abstractions for concurrency.
 Using these abstractions, Pekko also provides libraries for persistence, streams, HTTP, and more.
 
+Pekko is part of the Apache Incubator program and we are actively working towards a v1.0.0 release. It's not yet ready for production use, but please get involved with the community and contribute to the effort.
+
 Pekko is a fork of [Akka](https://github.com/akka/akka) 2.6.x, prior to the Akka project's adoption of the Business Source License.
 
 Reference Documentation
 -----------------------
 
-**TODO add documentation links**
+The reference documentation is available at [pekko.apache.org](https://pekko.apache.org/).
+
+Release tracking
+----------------
+ 
+The Pekko project spans multiple Github repositories. Work required for a whole Pekko project release (multiple repositories) is tracked in the [Apache Pekko Github project](https://github.com/orgs/apache/projects/220/views/1). Note that due to Apache Software Foundation regulations, this cannot be made public and is viewable only by Apache project [committers](https://www.apache.org/foundation/how-it-works.html#committers). Therefore non-committers check up on the Github Discussions and dev mailing list to confirm contents of the current in-development release.
+
+
+Repositories
```

Comment:
I generally don't agree with most of the changes here and kind of think that this stuff is better driven via issues or email discussion than PRs.

I definitely don't think that the README in this Git repo is our main doc. Our main doc is https://pekko.apache.org

I have taken this 'repositories' section and included it at https://pekko.apache.org/modules.html

Reply:
If we are specifically dealing with contributing, the convention is to put a cheat sheet/quick start guide in `CONTRIBUTING.md` in the root git repo (this convention is also followed by github, i.e. it will pick it up and reference it when someone tries to make a PR).

Of course within that `CONTRIBUTING.md` you can always reference to the actuals docs which will be hosted on the main `https://pekko.apache.org/` site.

Reply:
With the small team and the large number of repos - duplicated docs are turning into a real maintenance nightmare. I have no problem linking to one source of truth.

Reply:
a PR merged today has added a link to the pekko.apache.org docs

Reply:
Thanks for the feedback. Right now, there is a conversation (resumed from a past discussion) in the mailing list, see https://lists.apache.org/thread/8gnbv8g8z9fjlnlgbp903f51f3fd8634. I'm happy to reopen a thread with the "[DISCUSS]" tag. Also, I agree with the "avoid duplication" point:
 - I'm proceeding by taking in the changes added yesterday to point to the actual documentation.
 - I'm proceeding by removing the "Repository" section, as it would lead to duplication.